nucleus/dataset.py

```diff
@@ -564,7 +564,9 @@ def iloc(self, i: int) -> dict:
                     }
                 }
         """
-        response = self._client.dataitem_iloc(self.id, i)
+        response = self._client.make_request(
```

Comment:
any particular reason we don't use the existing `dataitem_iloc` anymore?

Reply:
Yes, it's deprecated.